models/classes/Platform/LtiPlatform.php

```diff
@@ -22,92 +22,94 @@
 
 namespace oat\taoLti\models\classes\Platform;
 
+use OAT\Library\Lti1p3Core\Platform\PlatformInterface;
+
 /**
  * LTI platform Value Object.
  */
-class LtiPlatform
+class LtiPlatform implements PlatformInterface
```

Comment:
Can you help me to understand why we need this custom implementation, please? 

There is an implementation already provided by the core lib.

Reply:
This implementation has additional properties and I could not extend the class from the library, it has private methods

Reply:
Let's just rename it while